core/src/main/java/org/apache/accumulo/core/data/ArrayByteSequence.java

```diff
@@ -180,4 +180,21 @@ public byte[] toArray() {
   public String toString() {
     return new String(data, offset, length, UTF_8);
   }
+
+  public void setArray(byte[] data, int offset, int length) {
```

Comment:
This needs javadoc with a `@since 3.1.0` tag.

Reply:
Done, updated.